src/store-api/src/region_request.rs

```diff
@@ -661,23 +663,138 @@ impl From<v1::ChangeColumnType> for ChangeColumnType {
 #[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize)]
 pub enum ChangeOption {
     TTL(Duration),
+    TwscMaxActiveWindowRuns(Option<usize>),
+    TwscMaxActiveWindowFiles(Option<usize>),
+    TwscMaxInactiveWindowRuns(Option<usize>),
+    TwscMaxInactiveWindowFiles(Option<usize>),
+    TwscMaxOutputFileSize(Option<ReadableSize>),
+    TwscTimeWindow(Option<Duration>),
+    TwscRemoteCompaction(Option<bool>),
+    TwscFallbackToLocal(Option<bool>),
+}
+
+impl ChangeOption {
+    pub fn to_change_table_option(&self) -> ChangeTableOption {
+        match self {
+            ChangeOption::TTL(duration) => ChangeTableOption {
+                key: "ttl".to_string(),
+                value: humantime::format_duration(*duration).to_string(),
+            },
+            ChangeOption::TwscMaxActiveWindowRuns(runs) => ChangeTableOption {
+                key: "compaction.twcs.max_active_window_runs".to_string(),
+                value: runs.map_or(String::new(), |value| value.to_string()),
+            },
+            ChangeOption::TwscMaxActiveWindowFiles(files) => ChangeTableOption {
+                key: "compaction.twcs.max_active_window_files".to_string(),
+                value: files.map_or(String::new(), |value| value.to_string()),
+            },
+            ChangeOption::TwscMaxInactiveWindowRuns(runs) => ChangeTableOption {
+                key: "compaction.twcs.max_inactive_window_runs".to_string(),
+                value: runs.map_or(String::new(), |value| value.to_string()),
+            },
+            ChangeOption::TwscMaxInactiveWindowFiles(files) => ChangeTableOption {
+                key: "compaction.twcs.max_inactive_window_files".to_string(),
+                value: files.map_or(String::new(), |value| value.to_string()),
+            },
+            ChangeOption::TwscMaxOutputFileSize(size) => ChangeTableOption {
+                key: "compaction.twcs.max_output_file_size".to_string(),
+                value: size.map_or(String::new(), |value| value.to_string()),
+            },
+            ChangeOption::TwscTimeWindow(window) => ChangeTableOption {
+                key: "compaction.twcs.time_window".to_string(),
+                value: window.map_or(String::new(), |value| {
+                    humantime::format_duration(value).to_string()
+                }),
+            },
+            ChangeOption::TwscRemoteCompaction(flag) => ChangeTableOption {
+                key: "compaction.twcs.remote_compaction".to_string(),
```

Comment:
Not sure if this option can be modified dynamically? @zyy

Reply:
I see i left this option out in this new update